presto-native-execution/presto_cpp/main/http/HttpServer.h

```diff
@@ -337,6 +354,27 @@ class HttpServer {
     registerDelete(pattern, endPointWrapper(callback));
   }
 
+  std::unordered_map<
```

Comment:
Any functional change in this file?

Reply:
No, just to expose endpoints from the server